lfs.h

```diff
@@ -136,7 +136,7 @@ enum lfs_open_flags {
     LFS_F_READING = 0x040000, // File has been read since last flush
     LFS_F_ERRED   = 0x080000, // An error occured during write
     LFS_F_INLINE  = 0x100000, // Currently inlined in directory entry
-    LFS_F_OPENED  = 0x200000, // File has been opened
+    LFS_F_OPENED  = 0x200000, // DEPRECATED: File has been opened
```

Comment:
Since these are marked as "internally used flags", I think it's safe to remove without deprecation.

Reply:
removed the change entirely